src/callbacks/onPreprocess/defineDataInsideTimeRange.js

```diff
@@ -0,0 +1,9 @@
+export default function defineDataInsideTimeRange() {
+    this.longDataInsideTimeRange = this.filtered_long_data.filter(
+        d => this.populationDetails.sampleInsideTimeRange.indexOf(d[this.config.id_col]) > -1
+    );
```

Comment:
Might be good to make a little wiki page that (briefly explains) the different data flows attached to the chart object. Otherwise it can be hard to keep track of them.